device/device.go

```diff
@@ -52,7 +51,7 @@ func Open(path string, options ...Option) (*Device, error) {
 	dev.cap = cap
 
 	// set preferred device buffer size
-	if reflect.ValueOf(dev.config.bufSize).IsZero() {
```

Comment:
Hi @alexandear the reflect package was used here to detect whether a value has never been set, hence test for its zero-value, or if it was set to the value zero.  But, looking back at the code, I think these tested values cannot have valid V4L2 zero values anyway, so this is OK.